components/Container.js

```diff
@@ -0,0 +1,32 @@
+
+import styled from 'styled-components/native'
+
+export const Container = styled.ImageBackground`
```

Comment:
Great idea to store reusable components in separate file.
Not sure though why this file is named `Container.js`, it has multiple components inside, not only `<Container />`